src/js/components/ConfigurationView.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import mixin from 'reactjs-mixin';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import DCOSStore from '../stores/DCOSStore'
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+
+class ConfigurationView extends mixin(StoreMixin) {
+  constructor() {
+    super();
+
+    this.store_listeners = [{name: 'dcos', events: ['change']}];
+  }
+
+  render() {
+    let {headline, service, versionID} = this.props;
+    let configuration = service.getVersions().get(versionID);
+
+    if (!configuration) {
+      DCOSStore.fetchServiceVersion(service.getId(), versionID);
```

Comment:
We should perform this dance in `componentDidMount` and update the internal state with whatever Service Version the DCOSStore emits when ready by listening to the appropriate events on the Store. 
The render function should be "pure" and just concern itself to render whatever is in the state, without side-effects.